.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    // Don't show the "Required assets are missing..." dialog on startup. We'll manage tasks.json / launch.json manually.
```

Comment:
>Required assets are missing... [](http://example.com/codeflow?start=23&length=30)

I think it was the dialog that generated a lot of changes to these files last time. :) Thanks for suppressing it!